src/views/AddTimer.tsx

```diff
@@ -0,0 +1,132 @@
+                </StyledLabel>
+                {(type === "countdown" || type === "stopwatch") && (
+                    <>
+                        <StyledLabel>
+                            Hours:
+                            <input name="hours" type="number" placeholder="Hours" value={config.hours} onChange={handleInput} />
+                        </StyledLabel>
+                        <StyledLabel>
+                            Minutes:
+                            <input name="minutes" type="number" placeholder="Minutes" value={config.minutes} onChange={handleInput} />
+                        </StyledLabel>
+                        <StyledLabel>
+                            Seconds:
+                            <input name="seconds" type="number" placeholder="Seconds" value={config.seconds} onChange={handleInput} />
+                        </StyledLabel>
+                    </>
+                )}
+                {type === "xy" && (
+                    <>
+                        <StyledLabel>
+                            Minutes:
+                            <input name="minutes" type="number" placeholder="Minutes" value={config.minutes} onChange={handleInput} />
+                        </StyledLabel>
+                        <StyledLabel>
+                            Seconds:
+                            <input name="seconds" type="number" placeholder="Seconds" value={config.seconds} onChange={handleInput} />
+                        </StyledLabel>
+                        <StyledLabel>
+                            Rounds:
+                            <input name="numberOfRounds" type="number" placeholder="Rounds" value={config.numberOfRounds} onChange={handleInput} />
+                        </StyledLabel>
+                    </>
+                )}
+                {type === "tabata" && (
+                    <>
+                        <StyledLabel>
+                            Work Time (seconds):
+                            <input name="workTime" type="number" placeholder="Work Time (seconds)" value={config.workTime} onChange={handleInput} />
+                        </StyledLabel>
+                        <StyledLabel>
+                            Rest Time (seconds):
+                            <input name="restTime" type="number" placeholder="Rest Time (seconds)" value={config.restTime} onChange={handleInput} />
+                        </StyledLabel>
+                        <StyledLabel>
+                            Rounds:
+                            <input name="numberOfRounds" type="number" placeholder="Rounds" value={config.numberOfRounds} onChange={handleInput} />
+                        </StyledLabel>
+                    </>
+                )}
+                <StyledButtonContainer>
+
+                    <Button
+                        type="submit"
+                        height={40}
+                        width={100}
+                        onClick={() => navigate("/")}
```

Comment:
I would not navigate back to the timer page after submission - It becomes inconvenient to add multiple timers, because you end up bouncing back and forth. I would reset the form so that the user can add the next timer.